src/containers/home/home.connector.js

```diff
@@ -1,9 +1,11 @@
 import { connect } from "react-redux";
 import Home from "./home.component";
 import { logout } from "store/thunks/users";
+import { getAllPostsToRender } from "store/thunks/posts";
 
 const mapDispatchToProps = (dispatch) => ({
   logoutThunk: () => dispatch(logout()),
+  getAllPostsToRenderThunk: () => dispatch(getAllPostsToRender()),
```

Comment:
A more apt name would be `fetchPostsThunk`. The thunk does not care whether you are going to render the posts. It's singular job is to fire a thunk that will fetch you the posts.

Reply:
done